include/rocksdb/utilities/env_mirror.h

```diff
@@ -33,9 +33,21 @@ class WritableFileMirror;
 
 class EnvMirror : public EnvWrapper {
   Env* a_, *b_;
+  bool free_a_, free_b_;
```

Comment:
There are no in-tree users of EnvMirror.  It's a debugging tool for those implementing new Env's.  Ceph uses it here: https://github.com/ceph/ceph/blob/master/src/os/bluestore/BlueStore.cc#L3044-L3061